### PR TITLE
[Hazmat Shipping] Add hazmat tracks

### PR DESCRIPTION
**Pull request by ThomazFB**
Summary
==========
Introduce a set of Tracks to the Shipping Label creation flow regarding Hazmat package declarations. The following tracks were added:

- CONTAINS_HAZMAT_CHECKED -> When the `Contains Hazardous Materials` toggle is activated
- HAZMAT_CATEGORY_SELECTOR_OPENED -> When the Hazmat category selector is displayed to the user
- HAZMAT_CATEGORY_SELECTED -> When the user selects an Hazmat category
   **Parameters**: 
   - Category: The selected category
   - Order ID: ID of the order with hazmat content under the Shipping Label creation process

How to Test
==========
1. Open the order details of an order eligible for Shipping label creation
2. Start the shipping label creation flow with valid US address options until you reach the Package details section
3. Inside the package details view, make sure the Contains hazardous materials toggle is visible and interactable
4. Hit the `Contains Hazardous Materials` toggle and verify that `🔵 Tracked: contains_hazmat_checked` is triggered.
5. Make sure the toggle expands and collapses a basic view that will serve as the main control options for the HAZMAT declaration
6. Hit the `Select a category` button and make sure the `🔵 Tracked: hazmat_category_selector_opened` is triggered.
7. Select a hazmat option and make sure the `🔵 Tracked: hazmat_category_selected` is triggered.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by ThomazFB** [comment]
@hafizrahman @JorgeMucientes, since since Alejandro will be unavailable until the end of the month, I'm requesting a review from you folks too, I hope that's OK! Thanks in advance 🙇

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/EditShippingLabelPackagesViewModel.kt [hunk 1]
```diff
@@ -203,6 +220,12 @@ class EditShippingLabelPackagesViewModel @Inject constructor(
         triggerEvent(OpenURL(url))
     }
 
+    fun onContainsHazmatChanged(isActive: Boolean) {
+        if (isActive) {
+            analyticsWrapper.track(AnalyticsEvent.CONTAINS_HAZMAT_CHECKED)
+        }
+    }
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/EditShippingLabelPackagesViewModel.kt:227]: It seems this function is never called when enabling/disabling the toggle. So the event `CONTAINS_HAZMAT_CHECKED` is never tracked.

**Review by JorgeMucientes — CHANGES_REQUESTED** [review]

**Comment by JorgeMucientes** [comment]
Good work @ThomazFB! I noticed that one of the events is never tracked and we should address that before merging. The rest of the events are working as expected 👍🏼

**Comment by ThomazFB** [comment]
Sorry @JorgeMucientes, I forgot to push yesterday's commits before going to the airport, which addresses the point you mentioned. It's OK now. Thank you for your review 🙇

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
No problem @ThomazFB. The PR is good to go now! :shipit:

**Pull request merged**